dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasableResourceInterface.java

```diff
@@ -49,27 +52,38 @@
      */
     T getAndCheckResourceByAlias(String alias);
 
-    default T addAliases(User user, Long id, String aliases) {
-        T c = getAndCheckResource(user, id);
-        Set<String> oldAliases = c.getAliases().keySet();
-        Set<String> newAliases = Sets.newHashSet(Arrays.stream(aliases.split(",")).map(String::trim).toArray(String[]::new));
-
+    default void checkAliases(Set<String>  aliases, User user) {
```

Comment:
Can you add a unit test for this method? In general, a PR without tests should be the exception.

Reply:
At a glance, I don't think this method uses `this` (but I might be wrong). If so, you could instead just make this a static method. It would make your testing easier, and you wouldn't have to pass a WorkflowResource to ZenodoHelper.createAliasUsingDOI.

Reply:
Thanks for the suggestion! Done